terraform/modules/network-validator/scripts/network_validation.sh

```diff
@@ -658,6 +662,21 @@ if ${debug}; then set -x ; fi
 # This way you can catch the error in case mysqldump fails in `mysqldump |gzip`, for example.
 set -o pipefail
 
+# Convert to lowercase for case insensitive check
+secure_mode=$(echo "$SECURE_MODE" | tr '[:upper:]' '[:lower:]')
+
+# Check if script needs  to be validated for secure production mode
+if [ "$secure_mode" = "true" ]; then
+    # Changing the required ports default value for secure production mode
+    if [[ -n ${IDCS_PORT} ]]
+    then
+      WLS_LB_PORT=${IDCS_PORT}
+    else
+      WLS_LB_PORT=7004
+    fi
+    ADMIN_HTTPS_PORT=9002
```

Comment:
This shouldn't be hard coded. A customer is able to set a different Administration Port in secured production mode.